### PR TITLE
Improve documentation for Numeric Scale

### PR DESCRIPTION
**Pull request by yjacolin**

### CODE REVIEW

**Comment by yjacolin** [comment]
@tsauerwein (have no write access to the repository)

File: docs/src/main/resources/templates/jasperreports.html [hunk 1]
```diff
@@ -524,7 +524,22 @@
 </p>
 <h4>6: others</h4>
 <p>
-Finally, other problems arise making the preview unusable.
+In order to add a numeric scale you have to create a Text field with the
```
Comment (tsauerwein) [inline docs/src/main/resources/templates/jasperreports.html]: 'text field' lowercase

File: docs/src/main/resources/templates/jasperreports.html [hunk 2]
```diff
@@ -524,7 +524,22 @@
...
 <p>
-Finally, other problems arise making the preview unusable.
+In order to add a numeric scale you have to create a Text field with the 
+following expression:
+</p>
+<div class="highlight"><pre>
+   String.format("Massstab 1: %.0f", $P{mapContext}.getRoundedScale())
+</pre></div>
+<p>
+Finally add the missing mapContext Parameter with the following properties:
```
Comment (tsauerwein) [inline docs/src/main/resources/templates/jasperreports.html]: 'parameter' lowercase

**Comment by tsauerwein** [comment]
Thanks for adding this!

**Comment by yjacolin** [comment]
@tsauerwein  done!

**Comment by tsauerwein** [comment]
Thanks a lot!

**Pull request merged**